Task10_Auth/src/main/java/reseponsesDTO/TokenDto.java

```diff
@@ -0,0 +1,40 @@
+package reseponsesDTO;
+
+public class TokenDto {
+    private String access_token;
+    private String token_type;
+    private int expires_in;
+    private String scope;
```

Comment:
Use camel case naming and @JsonProperty for mapping in this case

Reply:
done